active_auvs:
  - bluerov2_0
  - bluerov2_1
  # - bluerov2_2
  # - bluerov2_3
  # - bluerov2_4
  # - bluerov2_5
  # - bluerov2_6

planners:
  update_freq: 1
  random_linear_vel_range: [-1,1]
  random_angular_vel_range: [-0.0,0.0]

bluerov2_0:
  planner: linear
  start_twist:
    x: 1.0
    y: 0.0
    z: 0
    psi: 0

bluerov2_1:
  planner: stopped
  start_twist:
    x: 1.0
    y: 0.0
    z: 0
    psi: 0

bluerov2_2:
  planner: stopped
  start_twist:
    x: 1.0
    y: 0.0
    z: 0
    psi: 0

bluerov2_3:
  planner: stopped
  start_twist:
    x: 0.0
    y: 0.0
    z: 0
    psi: 0

bluerov2_4:
  planner: stopped
  start_twist:
    x: 0.0
    y: 0.0
    z: 0
    psi: 0

bluerov2_5:
  planner: stopped
  start_twist:
    x: 0.0
    y: 0.0
    z: 0
    psi: 0
  